src/components/LoginPanel.tsx

```diff
@@ -9,13 +9,32 @@ import styled, { useTheme } from "styled-components";
 
 const MiddleDiv = styled.div`
     max-width: 400px;
-    border: solid 1px ${(props) => props.theme.foregroundColor};
+    box-shadow: 0px 0px 15px ${(props) => props.theme.shadow};
     border-radius: 10px;
     padding: 50px;
     margin: auto;
+    background-color: ${(props) => props.theme.main.background[0]};
 
-    & button:hover {
-        color: ${(props) => props.theme.secondaryForegroundColor};
+    --fonts-buttonFontFamily: "Inter-Regular", san-serif;
+    --fonts-bodyFontFamily: "Inter-Regular", san-serif;
+    --fonts-inputFontFamily: "Inter-Regular", san-serif;
+    --fonts-labelFontFamily: "Inter-Bold", san-serif;
+
+    & input {
+        border-color: ${(props) => props.theme.main.background[0]};
+        background-color: ${(props) => props.theme.main.background[1]};
+    }
+
+    & input:-webkit-autofill,
+    & input:-webkit-autofill:hover,
+    & input:-webkit-autofill:focus,
+    & input:-webkit-autofill:active {
```

Comment:
is this compatible with non chromium/safari browsers? Styled components also does automatic vendor prefixing.

Have a look at these links for info on what I'm talking about:
https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions
https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix

Reply:
This is a Google-Chrome-specific implementation
The main reason is Google Chrome auto password fill makes the block have a blue background which is incompatible to the green theme. This chunk is just to disable that

Reply:
Done